packages/admin-next/dashboard/src/v2-routes/promotions/promotion-create/promotion-create.tsx

```diff
@@ -0,0 +1,46 @@
+import { RouteFocusModal } from "../../../components/route-modal"
+import { useCampaigns } from "../../../hooks/api/campaigns"
+import {
+  usePromotionRuleAttributes,
+  usePromotionRuleOperators,
+  usePromotionRules,
+} from "../../../hooks/api/promotions"
+import { CreatePromotionForm } from "./components/create-promotion-form/create-promotion-form"
+
+export const PromotionCreate = () => {
+  const { attributes: ruleAttributes } = usePromotionRuleAttributes("rules")
+  const { attributes: targetRuleAttributes } =
+    usePromotionRuleAttributes("target-rules")
+  const { attributes: buyRuleAttributes } =
+    usePromotionRuleAttributes("buy-rules")
+
+  const { rules } = usePromotionRules(null, "rules")
+  const { rules: targetRules } = usePromotionRules(null, "target-rules")
+  const { rules: buyRules } = usePromotionRules(null, "buy-rules")
+  const { operators } = usePromotionRuleOperators()
+  const { campaigns } = useCampaigns()
+
+  return (
+    <RouteFocusModal>
```

Comment:
todo: there is x-axis scrollbar visible especially stands out when you turn on dark mode